src/test/java/bio/terra/integration/AzureIntegrationTest.java

```diff
@@ -453,8 +455,14 @@ private void populateOmopTable() throws Exception {
   @Test
   public void testSnapshotCreateFromRequest() throws Exception {
     populateOmopTable();
-    UUID snapshotRequestId = makeSnapshotAccessRequest().getId();
-    SnapshotSummaryModel snapshotSummaryByRequest = makeSnapshotFromRequest(snapshotRequestId);
+
+    SnapshotAccessRequestResponse snapshotAccessRequest = makeSnapshotAccessRequest();
+    snapshotAccessRequest.status(SnapshotAccessRequestStatus.APPROVED);
+    snapshotAccessRequest.statusUpdatedDate(String.valueOf(Instant.now()));
```

Comment:
Could you do something like this?
```suggestion
    SnapshotAccessRequestResponse snapshotAccessRequest = makeSnapshotAccessRequest().status(SnapshotAccessRequestStatus.APPROVED).statusUpdatedDate(String.valueOf(Instant.now()));
```